pkg/apis/migration/migration.go

```diff
@@ -401,21 +401,21 @@ func (volumeMigration *volumeMigration) registerVolume(ctx context.Context, volu
 		createSpec.BackingObjectDetails = &cnstypes.CnsBlockBackingDetails{BackingDiskUrlPath: backingDiskURLPath}
 		log.Infof("Registering volume: %q using backingDiskURLPath :%q", volumeSpec.VolumePath, backingDiskURLPath)
 		log.Debugf("vSphere CSI driver registering volume %q with create spec %+v", volumeSpec.VolumePath, spew.Sdump(createSpec))
-		volumeID, err = (*volumeMigration.volumeManager).CreateVolume(ctx, createSpec)
+		volumeInfo, err = (*volumeMigration.volumeManager).CreateVolume(ctx, createSpec)
 		if err != nil {
 			log.Warnf("failed to register volume %q with createSpec: %v. error: %+v", volumeSpec.VolumePath, createSpec, err)
 		} else {
 			break
 		}
 	}
-	if volumeID != nil {
-		log.Infof("Successfully registered volume %q as container volume with ID: %q", volumeSpec.VolumePath, volumeID.Id)
+	if volumeInfo != nil {
```

Comment:
Can we also add a check for `volumeInfo.VolumeId` is not nil.

Reply:
This is not possible CreateVolume in manager.go return volumeInfo.VolumeId to be nil when volumeInfo is not nil. "VolumeId" is not a pointer type.